WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -293,6 +324,35 @@ extension SiteInformationWizardContent: InlineEditableNameValueCellDelegate {
     }
 }
 
+extension SiteInformationWizardContent: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        for (index, row) in Row.allCases.enumerated() {
+            guard let rowTextField = valueTextField(for: row) else {
+                let errorMessage = "We expect all rows to have `valueTextField`.  Please review the logic."
+                CrashLogging.logMessage(errorMessage, properties: nil, level: .error)
+                assertionFailure(errorMessage)
+                continue
+            }
+
+            guard rowTextField == textField else {
```

Comment:
Could we bring this guard up to `guard let rowTextField = valueTextField(for: row)` ?

Reply:
The problem with doing that is that this case shouldn't trigger an error message, while the case above should.

Reply:
👍